### PR TITLE
update codeowners for natlab

### PR DESCRIPTION
**Pull request by gytsto**
### Description
add myself to nat-lab codeowners

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean
- [x] README.md is updated
- [x] changelog.md is updated

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,3 @@
 * @NordSecurity/libtelio-team
 .github/workflows @Jauler @jjanowsk @packgron @tomaszklak
+nat-lab/* @gytsto
```
Comment (tomaszklak) [inline CODEOWNERS:3]: If I'm reading correctly the [documentation example](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax):
```
# In this example, @doctocat owns any files in the build/logs
# directory at the root of the repository and any of its
# subdirectories.
/build/logs/ @doctocat

# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
docs/*  docs@example.com
```
what you did will only make you an owner of files directly in the nat-lab directory but not for files deeper. I think you just want:
```
nat-lab/ @gysto
```

Comment (jjanowsk) [reply CODEOWNERS:3]: or just
```
nat-lab @gysto
```
to match style with worklows

Comment (gytsto) [reply CODEOWNERS]: alrighty, thanks for catching my bs... updated :smile:

**Review by tomaszklak — APPROVED** [review]
+2

**Pull request merged**